tests/php/sync/test_class.jetpack-sync-full.php

```diff
@@ -428,7 +431,7 @@ function test_full_sync_doesnt_sends_forbiden_private_or_public_post_meta() {
 		$this->assertEquals( null, $this->server_replica_storage->get_metadata( 'post', $post_id, 'snapTW', true ) );
 		$this->assertEquals( 'foo4', $this->server_replica_storage->get_metadata( 'post', $post_id, '_wp_attachment_metadata', true ) );
 		$this->assertEquals( 'foo5', $this->server_replica_storage->get_metadata( 'post', $post_id, 'a_public_meta', true ) );
-
+		Jetpack_Sync_Settings::update_settings( array( 'post_meta_whitelist' => array( ) ) );
```

Comment:
Is this line necessary for the tests to pass? If so we have an issue with how we're resetting data.